ui/appui-abstract/src/appui-abstract/quantity/BaseQuantityDescription.ts

```diff
@@ -20,10 +20,13 @@ export abstract class BaseQuantityDescription implements PropertyDescription {
   public displayLabel: string;
   public typename: string;
   public editor: PropertyEditorInfo;
+  public kindOfQuantityName: string;
 
-  constructor(name: string, displayLabel: string, iconSpec?: string) {
+
+  constructor(name: string, displayLabel: string, kindOfQuantityName: string, iconSpec?: string) {
```

Comment:
This breaks the API - the third argument was `iconSpec`, and now it's `kindOfQuantityName`. And I would say it breaks it in a very bad way, because the types of the two arguments are the same, so if anyone was passing a value for `iconSpec` - there will be no build error, but the value will now be set on `kindOfQuantityName` rather than what it was previously set on.

Same with changes in `FormattedQuantityDescription`.